From d52432a9bb0583e9bfd8cabbfdc5de60aa185099 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Thu, 16 Feb 2017 20:15:55 -0500
Subject: [PATCH 126/134] event-parse: Change pevent_data_pc() to
 pevent_data_preempt_count()

This function is for external use outside of trace-cmd itself. Instead of
using the cryptic "pc" extend the name to be more meaningful, preempt_count.

Signed-off-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
---
 event-parse.c | 4 ++--
 event-parse.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/event-parse.c b/event-parse.c
index 8206227866dc..32171310bf82 100644
--- a/event-parse.c
+++ b/event-parse.c
@@ -5204,13 +5204,13 @@ int pevent_data_pid(struct pevent *pevent, struct pevent_record *rec)
 }
 
 /**
- * pevent_data_pc - parse the preempt count from the record
+ * pevent_data_preempt_count - parse the preempt count from the record
  * @pevent: a handle to the pevent
  * @rec: the record to parse
  *
  * This returns the preempt count from a record.
  */
-int pevent_data_pc(struct pevent *pevent, struct pevent_record *rec)
+int pevent_data_preempt_count(struct pevent *pevent, struct pevent_record *rec)
 {
 	return parse_common_pc(pevent, rec->data);
 }
diff --git a/event-parse.h b/event-parse.h
index 3f5bb6c0b970..34f093d020f6 100644
--- a/event-parse.h
+++ b/event-parse.h
@@ -711,7 +711,7 @@ void pevent_data_lat_fmt(struct pevent *pevent,
 int pevent_data_type(struct pevent *pevent, struct pevent_record *rec);
 struct event_format *pevent_data_event_from_type(struct pevent *pevent, int type);
 int pevent_data_pid(struct pevent *pevent, struct pevent_record *rec);
-int pevent_data_pc(struct pevent *pevent, struct pevent_record *rec);
+int pevent_data_preempt_count(struct pevent *pevent, struct pevent_record *rec);
 int pevent_data_flags(struct pevent *pevent, struct pevent_record *rec);
 const char *pevent_data_comm_from_pid(struct pevent *pevent, int pid);
 struct cmdline;
-- 
2.5.5

